### PR TITLE
Switch: Fix aria attributes

### PR DESCRIPTION
**Pull request by marker-dao**

### CODE REVIEW

File: packages/devextreme/js/ui/switch.js [hunk 1]
```diff
@@ -316,17 +316,17 @@ const Switch = Editor.inherit({
         });
     },
 
-    _renderValue: function() {
+    _renderValue() {
         this._validateValue();
 
-        const val = this.option('value');
-        this._renderPosition(val, 0);
+        const { value } = this.option();
```
Comment (EugeniyKiyashko) [inline packages/devextreme/js/ui/switch.js]: I think this approach can affect on Knockout integration. Perhaps, you will remember: [link](https://teams.microsoft.com/l/message/19:d52fea9c6ac94ea1abc470486f4a963a@thread.skype/1686051486996?tenantId=e4d60396-9352-4ae8-b84c-e69244584fa4&groupId=8c31b3e5-5870-46b1-92a7-180d13b282ed&parentMessageId=1686051486996&teamName=DevExtreme&channelName=Components%20Squad%20(English)&createdTime=1686051486996)

File: packages/devextreme/testing/tests/DevExpress.ui.widgets.editors/switch.markup.tests.js [hunk 2]
```diff
@@ -157,27 +157,59 @@ QUnit.module('Switch markup', () => {
...
+        instance.option({ value: true });
 
-        instance.option('value', false);
-        assert.equal($element.attr('aria-label'), 'off test', 'aria \'off state\' label is correct');
-        assert.equal($element.attr('aria-pressed'), 'false', 'aria \'off state\' pressed attribute is correct');
+        assert.strictEqual($element.attr('aria-label'), 'on test');
     });
-});
 
+    QUnit.test('Switch should have correct aria-disabled and tabindex attributes', function(assert) {
```
Comment (EugeniyKiyashko) [inline packages/devextreme/testing/tests/DevExpress.ui.widgets.editors/switch.markup.tests.js:192]: Should we split such tests into tests checking attributes on the default render and tests checking attributes after the option changes at runtime?

File: packages/devextreme/testing/tests/DevExpress.ui.widgets.editors/switch.markup.tests.js [hunk 3]
```diff
@@ -157,27 +157,59 @@ QUnit.module('Switch markup', () => {
...
-        assert.equal($element.attr('role'), 'button', 'aria role is correct');
+        assert.strictEqual($element.attr('role'), 'switch');
     });
 
-    QUnit.test('aria properties', function(assert) {
+    QUnit.test('Switch should have correct aria-checked attribute', function(assert) {
+        const $element = $('#switch').dxSwitch();
+        const instance = $element.dxSwitch('instance');
+
+        assert.strictEqual($element.attr('aria-checked'), 'false');
```
Comment (EugeniyKiyashko) [inline packages/devextreme/testing/tests/DevExpress.ui.widgets.editors/switch.markup.tests.js]: If test fails I don't have any comment what does mean each assert.

File: packages/devextreme/testing/tests/DevExpress.ui.widgets.editors/switch.markup.tests.js (hunk 2 above)
Comment (marker-dao) [reply packages/devextreme/testing/tests/DevExpress.ui.widgets.editors/switch.markup.tests.js:192]: No

File: packages/devextreme/js/ui/switch.js (hunk 1 above)
Comment (marker-dao) [reply packages/devextreme/js/ui/switch.js]: [389c8403d64d10319b382630250b40d2cf9e5d06]

File: packages/devextreme/testing/tests/DevExpress.ui.widgets.editors/switch.markup.tests.js (hunk 3 above)
Comment (marker-dao) [reply packages/devextreme/testing/tests/DevExpress.ui.widgets.editors/switch.markup.tests.js]: [389c8403d64d10319b382630250b40d2cf9e5d06]

**Review by EugeniyKiyashko — APPROVED** [review]

File: packages/devextreme/testing/tests/DevExpress.ui.widgets.editors/switch.markup.tests.js [hunk 4]
```diff
@@ -157,27 +157,59 @@ QUnit.module('Switch markup', () => {
...
+
+    QUnit.test('Switch should have correct aria-readonly attribute', function(assert) {
+        const $element = $('#switch').dxSwitch();
+        const instance = $element.dxSwitch('instance');
+
+        assert.strictEqual($element.attr('aria-readonly'), undefined, 'The element should not have aria-readonly attribute if readOnly is false');
+
+        instance.option({ readOnly: true });
+
+        assert.strictEqual($element.attr('aria-readonly'), 'true', 'The element must have aria-readonly=true if readOnly is true');
```
Comment (ksercs) [inline packages/devextreme/testing/tests/DevExpress.ui.widgets.editors/switch.markup.tests.js:213]: JFYI: Actually it's not necessary to write so long messages here.
Test title is for describing a scenario, assert message is for describing a checked state.

**Review by ksercs — APPROVED** [review]

**Pull request merged**